pkgs/applications/networking/mailreaders/electron-mail/default.nix

```diff
@@ -1,13 +1,13 @@
-{ appimageTools, lib, fetchurl }:
+{ appimageTools, lib, fetchurl, libsecret }:
 
-let
+stdenv.mkDerivation rec {
```

Comment:
You probably want to keep this line like it is.

Reply:
sure thing, i missed that during rebase